components/has/subscription.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: has-operator
+  namespace: application-service
+spec:
+  channel: alpha
+  installPlanApproval: Automatic
```

Comment:
We'll need to take a call on this in general.
https://github.com/redhat-appstudio/infra-deployments/issues/14

Reply:
and you are not alone :) 
https://github.com/redhat-appstudio/infra-deployments/blob/main/components/build/openshift-pipelines/subscription.yaml